tests/unit/test_dao.py

```diff
@@ -73,6 +73,11 @@ async def _get_dao(
         """*To be implemented by the provider. Input validation is done outside of this
         method.*
         """
+        ...
+
+        """*To be implemented by the provider. Input validation is done outside of this
```

Comment:
Is this duplicated on purpose?

Reply:
That was already there, fixed that in the other branch and forgot it here...

Reply:
Removed in 8a4c959a424537f0e49958f2e10ca1838bbd5701